vagrant/provisioning/roles/pentaho-configuration/tasks/main.yml

```diff
@@ -286,6 +286,21 @@
     delay: 5
     timeout: 300
 
+- name: restart pentaho if status 404
+  block:
+    - name: check for 404 pentaho
+      uri:
+        url: https://localhost:2002/pentaho
+        validate_certs: false
+      register: pentaho_status
+      ignore_errors: yes
+    - name: restart pentaho if status code 404
+      become: yes
+      systemd:
+        name: pentaho
+        state: restarted
+      when: pentaho_status.status == 404
```

Comment:
Should we again wait for Pentaho port to be available after restarting?

Reply:
Don't have a strict answer for this. Not really, because it has plenty retries on the next tasks and tried it against several playbooks(dev, ce, all) where waiting was not needed, and yes, we should wait again for Pentaho port to be available just for pure convenience.